app/Main.hs

```diff
@@ -157,16 +164,24 @@ showInput :: Input -> Text
 showInput Stdin = "(input)"
 showInput (File fp) = pack fp
 
+infixl 3 <??>
+(<??>) :: Maybe a -> a -> a
+(<??>) = flip fromMaybe
```

Comment:
Note, we already have this defined in `Swarm.Util` as `(?)`. (But `infixr 1` instead of `infixl 3`.)

Reply:
Ah, I saw Relude has [`(?:)`](https://hackage.haskell.org/package/relude-1.2.0.0/docs/Relude-Monad-Maybe.html#v:-63-:) as `infixr 0` but I did not know we have some operator too, thanks! 👍